### PR TITLE
Added integration for DragAndDrop sample with new screen manager SDK

### PR DESCRIPTION
**Pull request by ancirja-m**

### CODE REVIEW

**Review by CesarValiente — APPROVED** [review]
LGTM! Thanks Andrei! Just the build problem has to be fixed.

File: DragAndDrop/src/main/java/com/microsoft/device/display/samples/draganddrop/fragment/FragmentExtensions.kt [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    get() = if (fragments.size > 0) {
+        fragments[fragments.size - 1]
+    } else {
+        null
+    }
+
+/**
+ * Removes all fragments from back stack.
+ */
+fun FragmentManager.removeAll() {
```
Comment (bimiron) [inline DragAndDrop/src/main/java/com/microsoft/device/display/samples/draganddrop/fragment/FragmentExtensions.kt:25]: Are these extensions used anywhere?

**Review by bimiron — APPROVED** [review]
Nice work!

File: DragAndDrop/src/main/java/com/microsoft/device/display/samples/draganddrop/fragment/FragmentExtensions.kt (hunk 1 above)
Comment (ancirja-m) [reply DragAndDrop/src/main/java/com/microsoft/device/display/samples/draganddrop/fragment/FragmentExtensions.kt]: No. I removed this file. Thanks.

**Pull request merged**